cdc/kv/client.go

```diff
@@ -361,24 +360,24 @@ func (c *CDCClient) divideAndSendEventFeedToRegions(
 func (c *CDCClient) singleEventFeed(
 	ctx context.Context,
 	span util.Span,
-	ts uint64,
-	meta *metapb.Region,
+	checkpoint uint64,
```

Comment:
hoe about `checkPointTs`?

Reply:
Yes. Note that checkpoint is a valid word, not two words, so we should use `checkpoint` instead of `checkPoint`.